01 - Navigation/01 - Navigate to something you know/Code/JetBrains/Customers/Customer.php

```diff
@@ -57,4 +57,4 @@ public function getName()
     public function celebrateBirthday() {
         echo 'Yay!';
     }
-} 
\ No newline at end of file
+}
```

Comment:
I did not take this personal. Sorry if I made it look this way. Having rethought this, I (even more) agree with you in that it would be better to introduce several changes to make things consistent **and** right, compared to just taking the easier route and adapt a small portion to what's already there. Now I'm thinking about improving my commits, but I am unsure on this...